.github/workflows/test.yml

```diff
@@ -3,7 +3,7 @@ name: test
 on:
   push:
     branches: [master]
-    ignore-tags: ['v*']
+    tags-ignore: ['v*']
```

Comment:
https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-ignoring-branches-and-tags